WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -201,6 +205,9 @@ private extension ProductFormActionsFactory {
             return product.product.categories.isNotEmpty
         case .tags:
             return product.product.tags.isNotEmpty
+        // Downloadable (only the core product types for downloadable files has been handled. custom product type would be handled later) products only.
```

Comment:
Custom product types are not able to handle downloadable files (it should be like that also on Android).
```suggestion
        // Downloadable  files. Only core product types for downloadable files are able to handle downloadable files.
```

Reply:
💯